search/query/disjunction.go

```diff
@@ -71,3 +71,23 @@ func (q *DisjuctionQuery) Searcher(rs index.Readers) (search.Searcher, error) {
 
 	return searcher.NewDisjunctionSearcher(len(rs), srs)
 }
+
+// Equal reports whether q is equivalent to o.
+func (q *DisjuctionQuery) Equal(o search.Query) bool {
+	inner, ok := o.(*DisjuctionQuery)
+	if !ok {
+		return false
+	}
+
+	if len(q.Queries) != len(inner.Queries) {
+		return false
+	}
+
+	// TODO: Should order matter?
```

Comment:
nah i think it's fine to not care about this

Reply:
To not care about order? We'll need to sort the queries in that case